custom-namespaces.yaml

```diff
@@ -610,10 +617,10 @@ firefox_accounts:
       type: table_view
       tables:
         - table: mozdata.analysis.wclouser_fxa_health_counts
-    health_db_counts:
-      type: table_view
-      tables:
-        - table: mozdata.analysis.wclouser_fxa_health_db_counts
+      measures:  
+        count_sum:
+          type: sum
+          sql: ${count}
```

Comment:
This is kind of an unrelated change, but we'll allow it. 😉

Reply:
😰